tests/test_features.py

```diff
@@ -0,0 +1,7 @@
+import pyvista
+
+
+def test_extrude():
+    arc = pyvista.CircularArc([-1, 0, 0], [1, 0, 0], [0, 0, 0])
+    mesh = pyvista.extrude(arc, [0, 0, 1])
+    mesh.plot()
```

Comment:
likewise, the `extrude` filter would only need to be tested in the `test_polydata.py` module

Reply:
fixed in e37c761748d03b0651c468bad6ec687cce8d2276